lib/utils/link.js

```diff
@@ -25,6 +25,24 @@ function linkIfExists (from, to, gently, cb) {
   })
 }
 
+function resolveIfSymlink (maybeSymlinkPath, cb) {
+  fs.lstat(path.resolve(maybeSymlinkPath), function (err, stat) {
```

Comment:
The `path.resolve` calls aren't doing anything here. `fs.lstat` and `fs.readlink` both accept relative arguments.